tests/models/test_flex_message.py

```diff
@@ -218,6 +218,7 @@ def test_image_component(self):
         arg = {
             'url': 'https://example.com/flex/images/image.jpg',
             'size': 'full',
+            'animated': False,
```

Comment:
I think `None` is better than `False` because key with `None` value will be ignored in json result.

https://github.com/line/line-bot-sdk-python/blob/f520f9c80382367ace78322ffc8d99cb9bd99f26/linebot/models/base.py#L84